### PR TITLE
Updated the README with usage examples as per feedback.

### LINKED ISSUE #41: Feedback for Milestone 2
Great job! There are just some minor formatting changes
- It's a good practice to add some examples and outputs for some of your functions in the README as well. This immediately highlights the usefulness of your package whenever a user opens your repo.
- `test_drafted_person()` missing docstring.
- Add dependencies using the `` notation. For example `numpy==1.17.2` and also link to numpy's website.
- Amazing logo design!

### PR DESCRIPTION
**Pull request by jnederlo**
This closes #41 and #45

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "pypuck"
-version = "0.2.1"
+version = "0.2.2"
 description = "Functions to access the publicly available but uncodumented NHL.com API"
 authors = ["jarvis Nederlof <jarvisnederlof@gmail.com>"]
```
Comment (chuusan) [inline pyproject.toml]: Could you add the rest of us to the authors?

**Review by chuusan — CHANGES_REQUESTED** [review]
great!

File: pyproject.toml (hunk 1 above)
Comment (jnederlo) [reply pyproject.toml]: Will do

**Review by chuusan — APPROVED** [review]
Great!

**Pull request merged**